src/thor/dao/release_dao.py

```diff
@@ -0,0 +1,179 @@
+import os
+import sqlalchemy as sa
+from . import config
+from models.models import Release
+
+from sqlalchemy.orm import sessionmaker
+from contextlib import contextmanager
+
+import logging
+
+from contextlib import contextmanager
+
+# Implements CRUD functions on the database.
+
+engine = sa.create_engine(config.RELEASE_DATABASE_URL)
+Session = sessionmaker(bind=engine)
+
+logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
+logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
+l = logging.getLogger(__name__)
+
+
+@contextmanager
+def session_scope():
+    """ Provides transactional scope around a series of operations. 
+    In other words, handles the messiness of sessions and commits, 
+    including rollback, so other modules don't have to. """
+
+    session = Session()
+    try:
+        yield session
+        session.commit()
+    except:
+        session.rollback()
+        raise
+    finally:
+        session.close()
+
+
+def manCreateRelease(id, version, result):

```

Comment:
I'd recommend changing the `id` parameter. There's an `id` that's a builtin in python and this is override that name.